Makefile

```diff
@@ -71,6 +71,13 @@ endif
 
 LIBXXH = libxxhash.$(SHARED_EXT_VER)
 
+XXHSUM_SPLIT_SRCS = programs/xxhsum/xsum_os_specific.c \
```

Comment:
try to define something like : 
`XXHSUM_DIR = programs/xxhsum`
so that it's easier to update the directory
(it's very frequently invoked)